template.yaml

```diff
@@ -47,6 +47,7 @@ Resources:
       CodeUri: email_sender/
       Handler: app.lambda_handler_sqs
       Runtime: python3.9
+      MemorySize: 512
```

Comment:
Is this minimum necessary requirement?

Reply:
It is not the minimum necessary. When I tested with 128 MB, it seemed less. It still worked. But the execution did not look concurrent enough. We can try with 256 with 10 concurrent threads and see if it is performant enough.